### PR TITLE
Added ui to manage tasks

### PR DESCRIPTION
**Pull request by wachterjohannes**
Add ui to manage tasks.

Basic PR: https://github.com/sulu/sulu/pull/3037

translations: https://docs.google.com/spreadsheets/d/1zjvdpUxOpv9JLuhLgpq6Z1ueRX3og9VJVjlVqMK766c/edit

### CODE REVIEW

File: src/Sulu/Bundle/AutomationBundle/Resources/public/dist/components/automation-tab/overlay/form.html [hunk 1]
```diff
@@ -0,0 +1,43 @@
+<div class="grid">
+    <div class="grid-row">
+        <div class="grid-col-6">
+            <label for="task-name" class="required m-bottom-10"><%= translations.taskName %></label>
+            <div id="task-name"
+                 data-aura-component="select@husky"
+                 data-aura-data='[{"id":"sulu_content.publish","title":"Publish"},{"id":"sulu_content.unpublish","title":"Unpublish"}]'
```
Comment (wachterjohannes) [inline src/Sulu/Bundle/AutomationBundle/Resources/public/dist/components/automation-tab/overlay/form.html:7]: only data for testing. will be fetched in one of the next PR.

**Comment by wachterjohannes** [comment]
@danrot

File: src/Sulu/Bundle/AutomationBundle/Resources/public/js/main.js [hunk 2]
```diff
@@ -9,7 +9,9 @@
 
 require.config({
     paths: {
-        suluautomation: '../../suluautomation/js'
+        suluautomation: '../../suluautomation/js',
+
+        'services/suluautomation/task-manager': '../../suluautomation/js/services/task-manager'
```
Comment (danrot) [inline src/Sulu/Bundle/AutomationBundle/Resources/public/js/main.js:14]: Is that really necessary? I thought that's only required when we have the a validation type, where the path doesn't match.

File: src/Sulu/Bundle/AutomationBundle/Resources/public/js/services/task-manager.js [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+        save: function(data) {
+            return Util.save(url({id: data.id || null}), !!data.id ? 'PUT' : 'POST', data);
+        },
+
+        deleteItems: function(ids) {
+            return Util.save(url({id: null}) + '?ids=' + ids.join(','), 'DELETE');
+        }
+    };
+
+    return new Manager();
```
Comment (danrot) [inline src/Sulu/Bundle/AutomationBundle/Resources/public/js/services/task-manager.js]: Any reason this "complicated" structure is required? Why not simply returning an object?

**Review by danrot — CHANGES_REQUESTED** [review]
Tried to test it, but I couldn't see any tab after registering the SuluAutomationBundle.

File: src/Sulu/Bundle/AutomationBundle/Resources/public/js/main.js (hunk 2 above)
Comment (wachterjohannes) [reply src/Sulu/Bundle/AutomationBundle/Resources/public/js/main.js:14]: i thought we decided to "normalize" the service ids in this structure

File: src/Sulu/Bundle/AutomationBundle/Resources/public/js/services/task-manager.js (hunk 3 above)
Comment (wachterjohannes) [reply src/Sulu/Bundle/AutomationBundle/Resources/public/js/services/task-manager.js]: not really its mainly copied (:

File: src/Sulu/Bundle/AutomationBundle/Resources/public/js/main.js (hunk 2 above)
Comment (danrot) [reply src/Sulu/Bundle/AutomationBundle/Resources/public/js/main.js:14]: Haven't heard about that yet... Have we already done that somewhere?

Comment (wachterjohannes) [reply src/Sulu/Bundle/AutomationBundle/Resources/public/js/main.js:14]: yes category, media, contact ... bundle

**Comment by wachterjohannes** [comment]
@danrot i have excluded the tab on the startpage but after discussion it doenst make sense. so you can review it now also on the startpage.

**Comment by danrot** [comment]
After selecting a valid date, the form still shows it as invalid:
image

Can still be added by a click on the ok button though.

**Comment by danrot** [comment]
There is no confirmation dialog when clicking on delete in the tab.

**Comment by danrot** [comment]
The datepicker is still blinking red for a short time after a value is assigned to it for the first time.

**Comment by wachterjohannes** [comment]
@danrot this problem also exists in the rest of the system. i will take a deeper look into this in an own PR (i think in husky).

**Comment by wachterjohannes** [comment]
@danrot i have added a todo in the main PR.

**Pull request merged**